internal/core/thirdparty/knowhere/CMakeLists.txt

```diff
@@ -11,8 +11,8 @@
 # or implied. See the License for the specific language governing permissions and limitations under the License.
 #-------------------------------------------------------------------------------
 
-set( KNOWHERE_VERSION b724230 )
-set( GIT_REPOSITORY  "https://github.com/zilliztech/knowhere.git")
+set( KNOWHERE_VERSION update_knowhere_version )
+set( GIT_REPOSITORY  "https://github.com/foxspy/knowhere.git")
```

Comment:
Correct this when the knowhere is ready.

Reply:
done